whitepaper/chapters/cryptography.tex

```diff
@@ -33,33 +33,34 @@ \subsection{Signing and Verification}
 
 Given a message $M$, private key $k$ and its associated public key \underline{A}, the following steps are taken to create a signature:
 \begin{align}
-H(k) &=(h_0, h_1,..., h_{511}) \\
-r &= H(h_{256},...,h_{511}, M) \text{ where the comma means concatenation} \\
+\hf(k) &=(h_0, h_1,..., h_{511}) \\
+r &= \hf(h_{256},...,h_{511}, M) \text{ where the comma means concatenation} \\
 R &= rB \\
-S &= (r + H(\underline{R}, \underline{A}, M)a) \: mod \: q \label{eq:cryptography:S}
+S &= (r + \hf(\underline{R}, \underline{A}, M)a) \: mod \: q \label{eq:cryptography:S}
 \end{align}
 
 Then $(\underline{R}, \underline{S})$ is the \nind{signature} for the message $M$ under the private key $k$.
 Note that only signatures where $S<q$ and $S>0$ are considered as valid \textbf{to prevent} the problem of \emph{signature malleability}\index{signature!malleability}.
 
 To verify the signature $(\underline{R}, \underline{S})$ for the given message $M$ and public key \underline{A}, the verifier checks $S<q$ and $S>0$ and then calculates
 \begin{align*}
-\tilde{R} = SB - H(\underline{R}, \underline{A}, M)A
+\tilde{R} = SB - \hf(\underline{R}, \underline{A}, M)A
 \end{align*}
 and verifies that
 \begin{equation}
 \tilde{R} = R \label{eq:cryptography:verifyR}
 \end{equation}
 If $S$ was computed as shown in \eqref{eq:cryptography:S} then
-$$SB = rB + (H(\underline{R}, \underline{A}, M)a)B = R + H(\underline{R}, \underline{A}, M)A$$
+$$SB = rB + (\hf(\underline{R}, \underline{A}, M)a)B = R + \hf(\underline{R}, \underline{A}, M)A$$
 so \eqref{eq:cryptography:verifyR} will hold.
 
 \subsection{Batch Verification}
 
 When lots of signatures have to be verified, a batch signature verification can speed up the process by about 80\%.
 \codenamespace uses the algorithm outlined in \cite{Bernstein2011}.
 Given a batch of $(M_i, A_i, R_i, S_i)$ where $(R_i, S_i)$ is the signature for the message $M_i$ with public key $A_i$,
-uniform distributed 128-bit independent random integers $z_i$ are generated and $H_i(R_i, A_i, M_i)$ is calculated.
+$H_i = \hf(R_i, A_i, M_i)$ is calculated.
```

Comment:
not sure if following is a bit clearer?

Given a batch of $(M_i, A_i, R_i, S_i)$ where $(R_i, S_i)$ is the signature for the message $M_i$ with public key $A_i$, let $H_i = \hf(R_i, A_i, M_i)$.
Additionally, assume a corresponding number of uniform distributed 128-bit independent random integers $z_i$ are generated.